required/latex-lab/latex-lab-table.dtx

```diff
@@ -740,65 +736,57 @@
 % \end{plugdecl}
 %
 %
+% \begin{plugdecl}{code}
+%    This socket takes a number, checks if is larger than one,
+%    checks if the colspan already attribute exists (we can't predefine an
+%    arbitrary number), and updates \cs{l_@@_cellattribute_tl}.
+%    \begin{macrocode}
+\NewSocketPlug{tagsupport/tbl/colspan}{code}
+  {
+    \int_compare:nNnT {#1}>{1}
+     {
+       \prop_get:NeNF \g__tag_attr_entries_prop
+          {colspan-\int_eval:n{#1}}
+          \l_@@_tmpa_tl
+          {
+            \__tag_attr_new_entry:ee
+              {colspan-\int_eval:n{#1}}
+              {/O /Table /ColSpan~\int_eval:n{#1}}
+          }
+       \tl_set:Ne \l_@@_cellattribute_tl
+          {colspan-\int_eval:n{#1}}
+     }
+  }
+%    \end{macrocode}
+%    TODO: move to tagpdf
```

Comment:
can you not move already?

Reply:
probably may in fact has happened (this is the pr before Ulrike worked on it in her branch).

Reply:
I added it now to tagpdf and will remove it then here later, when tagpdf has been updated.